roxene/__init__.py

```diff
@@ -2,11 +2,6 @@
 
 PRECISION = tf.dtypes.float16
 
-from .Gene import Gene
-from .Organism import Organism
-from .Cell import Cell
-from .Neuron import Neuron
-from .InputCell import InputCell
-from .ConnectNeurons import ConnectNeurons
-from .ConnectNeurons import ConnectNeurons
-from .CreateNeuron import CreateNeuron
\ No newline at end of file
+from .core import Gene, Organism, Cell, InputCell
+from .neuron import Neuron
+from .genes import CreateNeuron, ConnectNeurons, RotateCells
```

Comment:
Taking a guess at the format of the file references and importing and stuff. I pieced it together, mostly from sugar packets.